README.md

```diff
@@ -41,10 +41,10 @@ PROXY_TO=http://localhost:5100
 You can run the tests with
 
 ```bash
-> yarn test
+> yarn test:vite
```

Comment:
if we're getting rid of jest in this PR, can't we update the `yarn test` script to use vite?

Reply:
Jest is bundled in with CRACO, it'll be removed at that point